src/main/java/com/iexec/core/replicate/ReplicatesController.java

```diff
@@ -21,6 +21,7 @@
 import com.iexec.common.replicate.*;
 import com.iexec.core.security.JwtTokenProvider;
 import com.iexec.core.worker.WorkerService;
+import feign.FeignException;
```

Comment:
```suggestion
```

Reply:
The dependency required after  @mcornaton recommended me to use a link in javadoc comment.
I could remove it by replacing {@link} with {@code} but the integration is not the same in IDE.

Reply:
Interesting :bulb: